scripts/wiki_ship_stat_parser.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import math
+import sys
+import copy
+import json
+import numpy as np
+import operator                 # for sorted
+import sys
+from pprint import pprint
+
+shiptype = {
```

Comment:
```
def GetShipType(s):
  return "%s [[File:%s.png]]" % (s.replace('_', ' ').capitalize(), s)
```

Reply:
True, but these strings are generated by hand "upstream", thus no  guartantee the patter will always be true. In fact, I don't know if it is true even now, as I haven't tested it for all ships/icons.

Reply:
Then I'd still keep the dict of exceptions, and use generated ones for the rest.